app/templates/components/navigation/nav-mobile.html

```diff
@@ -0,0 +1,41 @@
+<div
+    id="nav-mobile"
+    class="navigation bottom-0 left-0 fixed bg-corporate-blue translate-y-0 transition-all duration-300 text-white w-full desktop:hidden z-10 {% if false %}translate-y-full{% endif %}"
+>
+    <div class="w-full shadow-bottom">
+        <div
+            class="toggle mx-auto px-mobile py-4 max-w-4xl h-full flex justify-between font-bold"
+        >
+            <div data-toggle="accordion" class="flex cursor-pointer">
+                <img
+                    class="w-4 mr-6"
+                    src="{{ url_for('static', path='images/align-left.svg') }}"
+                    alt="{{ translations.nav.alignIconAlt }}"
+                />
+                <span class="font-bold text-sm w-full">
+                    {{ translations.nav.label }}
+                </span>
+            </div>
+            <img
+                data-toggle="accordionCloseOnly"
+                class="w-4 cursor-pointer hidden"
+                src="{{ url_for('static', path='images/x-circle.svg') }}"
+                alt="{{ translations.nav.alignIconAlt }}"
+            />
```

Comment:
This `<img>` should also be wrapped in a `<button>` (and you could add something like `aria-label="Inhaltsverzeichnis schließen"` to the button, and remove the `alt` tag from the `<img>`).